core/vm/interpreter.go

```diff
@@ -56,11 +56,11 @@ func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 	// If jump table was not initialised we set the default one.
 	var table *JumpTable
 	switch {
-	case evm.chainRules.IsPrague:
-		// TODO replace with prooper instruction set when fork is specified
-		table = &pragueInstructionSet
 	case evm.chainRules.IsCancun:
 		table = &cancunInstructionSet
+	case evm.chainRules.IsPrague:
+		// TODO replace with proper instruction set when fork is specified
+		table = &shanghaiInstructionSet
```

Comment:
This is an important change for replay since we're replaying history without many gas changes for Verkle.

Before potentially merging this branch into `kaustinen-with-shapella` we should find a reasonable way to configure this depending if the branch is used for Kaustinen or replay.